macrosynergy/dataquery/api.py

```diff
@@ -610,13 +659,41 @@ def download(self, tickers=None, xcats=None, cids=None, metrics=['value'],
         """
 
         if (cids is None) & (xcats is not None):
-            cids_dmca = ['AUD', 'CAD', 'CHF', 'EUR', 'GBP', 'JPY', 'NOK', 'NZD', 'SEK',
-                         'USD']  # DM currency areas
-            cids_dmec = ['DEM', 'ESP', 'FRF', 'ITL', 'NLG']  # DM euro area countries
-            cids_latm = ['BRL', 'COP', 'CLP', 'MXN', 'PEN']  # Latam countries
-            cids_emea = ['HUF', 'ILS', 'PLN', 'RON', 'RUB', 'TRY', 'ZAR']  # EMEA countries
-            cids_emas = ['CZK', 'CNY', 'IDR', 'INR', 'KRW', 'MYR', 'PHP', 'SGD', 'THB',
-                         'TWD']  # EM Asia countries
+            cids_dmca = [
+                "AUD",
+                "CAD",
+                "CHF",
+                "EUR",
+                "GBP",
+                "JPY",
+                "NOK",
+                "NZD",
+                "SEK",
+                "USD",
+            ]  # DM currency areas
+            cids_dmec = ["DEM", "ESP", "FRF", "ITL", "NLG"]  # DM euro area countries
+            cids_latm = ["BRL", "COP", "CLP", "MXN", "PEN"]  # Latam countries
+            cids_emea = [
```

Comment:
All python black recommendations included except this one - seems a bit unnecessary.